### DISCUSSION TITLE
Can't get to discussions from a discussion reply

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44231] [Mobile] [answered]
GitHub Mobile beta 
If one reaches a discussion from a notification, there's no way to get back to discussions for that repository.

The notification could be from Android notifications or from GitHub Mobile's notifications list, but it can take me here:

![Screenshot_20230113-095756.png](https://user-images.githubusercontent.com/2119212/212351528-ee08e085-0c76-4287-9cd1-5e8ae05156d2.png)

I can get to the post (which has a number that's a link), or to the repository (which has a link), but I can't directly get back to the discussions section or that discussion's category. 

For reference, on Chrome Android, you can get to both the discussion category and discussions in general:
![Screenshot_20230113-103335](https://user-images.githubusercontent.com/2119212/212358207-5d77d605-0a0d-4865-8574-cdb19eb37e7a.png)

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref, does the back button not work here?

**Reply by jsoref** [reply]
Two backs takes me here:

![Screenshot_20230117-100756.png](https://user-images.githubusercontent.com/2119212/212934793-e09ae850-10fc-4b31-84d2-9c86caa70d43.png)

Which is past discussions. Unlike in a browser. Then I have to try to get back to the right discussion/ category.

**Reply by stevepopovich** [reply]
Sorry, does 1 back press get you to where you expect?

**Reply by jsoref** [reply]
- First back just gets me to the discussion itself, e.g. https://github.com/community/community/discussions/44231

There's nothing that takes me from 

- https://github.com/community/community/discussions/44231#discussioncomment-4709265 to 
https://github.com/orgs/community/discussions/categories/mobile
or
https://github.com/orgs/community/discussions/

Which are both things I can reach from the browser (both mobile and desktop views)

**Reply by stevepopovich** [reply]
From the discussion screen header, you tap on the `Mobile` category to get to https://github.com/orgs/community/discussions/categories/mobile

You can also tap `community` in the top left part of the header to get to https://github.com/orgs/community/ which you can easily reach https://github.com/orgs/community/discussions/

**Reply by jsoref** [reply]
That's way less efficient than the path in the browser. And it's really roundabout.

**Reply by stevepopovich** [reply]
Seems to be a similar number of clicks. Thanks for the feedback though! We will consider this.
